bin/akula.rs

```diff
@@ -221,29 +226,54 @@ fn main() -> anyhow::Result<()> {
                                 .unwrap();
 
                             let mut api = Methods::new();
-                            api.merge(
-                                EthApiServerImpl {
-                                    db: db.clone(),
-                                    call_gas_limit: 100_000_000,
-                                }
-                                .into_rpc(),
-                            )
-                            .unwrap();
-                            api.merge(NetApiServerImpl { network_id }.into_rpc())
-                                .unwrap();
-                            api.merge(ErigonApiServerImpl { db: db.clone() }.into_rpc())
+
+                            let api_options: HashSet<String> = HashSet::from_iter(
+                                opt.enable_api
+                                    .into_iter()
+                                    .map(|s| s.to_lowercase())
+                                    .collect::<Vec<String>>(),
+                            );
+
+                            if api_options.get("eth").is_some() {
```

Comment:
```suggestion
                            if api_options.contains("eth") {
```

here and below

Reply:
Sounds good, I will change this now as well. Can you explain why using `!api_options.is_empty() && api_options.contains("eth")` is better than just `api_options.get("eth").is_some()`?

Reply:
Because `.contains` is the idiomatic usage of HashSet API and because we need to handle the case where `--enable_api` is not specified or is empty.